open-install-test.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Environment checks
+if [[ -z "${NEW_RELIC_API_KEY}" ]]; then
+  echo "Please set the NEW_RELIC_API_KEY environment variable"
+  exit 1
+fi
+if [[ -z "${NEW_RELIC_ACCOUNT_ID}" ]]; then
+  echo "Please set the NEW_RELIC_ACCOUNT_ID environment variable"
+  exit 1
+fi
+if [[ -z "${NEW_RELIC_REGION}" ]]; then
+  echo "Please set the NEW_RELIC_REGION environment variable"
+  exit 1
+fi
+
+# Check if we have the open install repo locally
+if [ ! -d "tmp/" ]
+then
+    mkdir tmp/ || true
+    pushd tmp
+    git clone git@github.com:newrelic/open-install-library.git
```

Comment:
alternatively, you can just use "make recipes" to fetch the latest recipe release.